.github/workflows/publish.yml

```diff
@@ -62,20 +62,41 @@ jobs:
           wget -qO ResotoCollect.json https://cdn.some.engineering/resoto/aws/edge/ResotoCollect.json
           wget -qO ResotoMutate.json https://cdn.some.engineering/resoto/aws/edge/ResotoMutate.json
 
-      - name: Update resource data models
+      - name: Wait for Docker images to build
+        if: github.event_name != 'workflow_dispatch'
+        uses: lewagon/wait-on-check-action@v1.2.0
+        with:
+          ref: ${{ github.ref }}
+          check-name: "Build split Docker images"
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Modify Docker Compose YAML
+        shell: bash
+        run: |
+          yq -i '(.services.resotoworker.environment += "RESOTOWORKER_OVERRIDE=resotoworker.collector=example")' docker-compose.yaml
+
+      - name: Clean existing resource data model images
         if: github.event_name == 'workflow_dispatch' # only when triggered manually
         shell: bash
+        run: |
+          cd ${{ github.workspace }}/resoto.com/static/img/kroki
+          find . -type f -name "*.svg" -prune -exec rm {} \;
```

Comment:
Making this `\+` instead of `\;` makes for a more efficient delete. It then only calls `rm` once, passing the list of all files that it should delete (more than once if the list of files is long). With `\;` it executes `rm` for every single file.